src/components/wells/Sample.vue

```diff
@@ -56,6 +63,7 @@ export default {
       outlier: false,
       warning: false,
```

Comment:
I would be tempted to turn this into an object because it does feel like they belong together. i.e.` warning: {}`. You could also add the properties in there if you want to provide documentation e.g. warning:` { message: null, shortMessage: null }`. It is better to set them to null as this will give a falsey value in your logic statement.

Reply:
Yes, I was considering having them as an object too, so I will do it 👍